### ISSUE TITLE
[Feature request]:  Include data on who opened a lock

### ISSUE THREAD

**Issue by iamehkri** [bropat/eufy-security-client#341]
### Describe the solution you'd like

For the Eufy R10/R20 smart lock there is a pin pad that you can assign access to within the app and when they open the door a notification is sent with their name.

I asked fuatakgun if this is a feature that can be implemented in his home assistant integration, he referred me to you in this thread:
https://github.com/fuatakgun/eufy_security/discussions/807

So basically an attribute showing who opened the lock.

Is this something that can be supported?

**Comment by InstigatorX** [comment]
+1 for this. Can use to automate actions in HA depending on if a code/fingerprint was used to unlock vs manually.
